test/ecs/ecs_metadata/resources/emf_prometheus_redis_schema.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Prometheus-Redis structured log schema",
+  "description": "JSON schema for the CloudWatch agent ECS structured log",
+  "type": "object",
+  "properties": {
+    "ClusterName": {},
+    "ECS_PROMETHEUS_EXPORTER_PORT": {},
+    "LaunchType": {},
+    "StartedBy": {},
+    "TaskClusterName": {},
+    "TaskDefinitionFamily": {},
+    "TaskGroup": {},
+    "TaskId": {},
+    "TaskRevision": {},
+    "Timestamp": {},
+    "Version": {},
+    "app_x": {},
+    "build_date": {},
+    "commit_sha": {},
+    "container_name": {},
+    "golang_version": {},
+    "instance": {},
+    "job": {},
+    "prom_metric_type": {},
+    "redis_exporter_build_info": {},
+    "version": {}
+  },
+  "required": [
+    "ClusterName",
```

Comment:
Does it need to require all of this value though? Only some is important here `prom_metric_type`, `instance`, `job` and related information of ECS. Other than that, I don't see any important dimensions required by the tests and also by the features.

Reply:
I took exactly what we emit and am validating that.